### PR TITLE
Golang page updates

### PR DESCRIPTION
**Pull request by iriberri**
Ready to review! :)

### CODE REVIEW

**Comment by iriberri** [comment]
Hi @plaindocs, I worked a bit more on this and I think this is mostly ready now (although I might start seeing typos/things to fix later, as it happened with the Python PR, hah!).

I deleted the whole mentions to `- core GNU build toolchain (autotools, make), cmake, scons` being available, since this is correctly documented in the different distribution pages.

File: user/languages/go.md [hunk 1]
```diff
@@ -43,11 +55,12 @@ go:
 
 All go version management is handled by [gimme](https://github.com/travis-ci/gimme).
 
-For precise versions pre-installed on the VM, please consult "Build system information" in the build log.
-
 ## Go Import Path
 
-The project source code will be placed in `GOPATH/src/github.com/user/repo` by default, but if [vanity imports](https://golang.org/cmd/go/#hdr-Remote_import_paths) are necessary (especially for [`internal` package imports](https://golang.org/cmd/go/#hdr-Internal_Directories)), `go_import_path:` may be specified at the top level of the config, e.g.:
+The project source code will be placed in `GOPATH/src/github.com/user/repo` by
```
Comment (plaindocs) [inline user/languages/go.md]: Does this mean anything code is copied there? Or that it should be placed there? Or ?

File: user/languages/go.md [hunk 2]
```diff
@@ -56,22 +69,25 @@ go_import_path: example.org/pkg/foo
 
 ## Dependency Management
 
-By default the install step defers to `go get ./...` or `go get -t ./...` if the version of go is greater than or equal
-to `1.2`.  If any of the following files are present, the default install step will be simply `true`:
+By default the install step defers to `go get ./...` or `go get -t ./...` if
```
Comment (plaindocs) [inline user/languages/go.md]: defers = defaults?

So, 

IF go >= 1.2 THEN go get -t ./..

IF go < 1.2 THEN go get ./..

IF makefile whatever THEN true

(and not make? )

**Comment by plaindocs** [comment]
@iriberri is this ready to merge?

File: user/languages/go.md (hunk 2 above)
Comment (iriberri) [reply user/languages/go.md]: Yes, that's it:

(for make vs go get) https://github.com/travis-ci/travis-build/blob/125c5d1fa8ada1f0d79bb77bbccaefd1e7e43c13/lib/travis/build/script/go.rb#L86-L91
and

(forgo get vs go get -t) https://github.com/travis-ci/travis-build/blob/125c5d1fa8ada1f0d79bb77bbccaefd1e7e43c13/lib/travis/build/script/go.rb#L146-L152

**Comment by iriberri** [comment]
I think it is ready, at least for a first iteration to update to the current format. I believe it'd be great to merge now and just iterate if needed.

**Pull request merged**